diff --git a/browser/components/moz.build b/browser/components/moz.build
index d16b27b1eaa3da7d11117731dfcf0903af71099a..aa8685c6c7db6209147ed08b628ae20fea70890b 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -28,6 +28,7 @@ with Files("controlcenter/**"):
 
 DIRS += [
     "about",
+    "abouthistory",
     "aboutlogins",
     "attribution",
     "contextualidentity",
@@ -53,6 +54,7 @@ DIRS += [
     "sessionstore",
     "shell",
     "syncedtabs",
+    "tabpreview",
     "uitour",
     "urlbar",
     "translation",
