diff --git a/browser/components/BrowserGlue.jsm b/browser/components/BrowserGlue.jsm
index 7bfc4ff7684931a656c0c1cd85a83082219e4f45..686272fe73e50f3c9dd9d6f7bb0639763230a038 100644
--- a/browser/components/BrowserGlue.jsm
+++ b/browser/components/BrowserGlue.jsm
@@ -7,6 +7,7 @@ var EXPORTED_SYMBOLS = [
   "BrowserGlue",
   "ContentPermissionPrompt",
   "DefaultBrowserCheck",
+  "JSWINDOWACTORS"
 ];
 
 const { XPCOMUtils } = ChromeUtils.import(
@@ -771,8 +772,8 @@ let JSWINDOWACTORS = {
     !Services.startup.showedPreXULSkeletonUI &&
     Services.prefs.getCharPref(
       "extensions.activeThemeID",
-      "default-theme@mozilla.org"
-    ) != "default-theme@mozilla.org"
+      Services.builtInThemes.DEFAULT_THEME_ID
+    ) != Services.builtInThemes.DEFAULT_THEME_ID
   ) {
     return;
   }
@@ -1237,7 +1238,8 @@ BrowserGlue.prototype = {
     os.addObserver(this, "handlersvc-store-initialized");
 
     ActorManagerParent.addJSProcessActors(JSPROCESSACTORS);
-    ActorManagerParent.addJSWindowActors(JSWINDOWACTORS);
+    /* This is now handled in dot/modules/glue.ts */
+    // ActorManagerParent.addJSWindowActors(JSWINDOWACTORS);
 
     this._flashHangCount = 0;
     this._firstWindowReady = new Promise(
@@ -1376,7 +1378,7 @@ BrowserGlue.prototype = {
 
     SessionStore.init();
 
-    BuiltInThemes.maybeInstallActiveBuiltInTheme();
+    // BuiltInThemes.maybeInstallActiveBuiltInTheme();
 
     if (AppConstants.MOZ_NORMANDY) {
       Normandy.init();
