diff --git a/toolkit/mozapps/extensions/internal/XPIProvider.jsm b/toolkit/mozapps/extensions/internal/XPIProvider.jsm
index 583512c02581dad6ded777a956f3dc4f3462065c..36b6872a6553ccf0ace833d1bbd9ca6be79bbc4b 100644
--- a/toolkit/mozapps/extensions/internal/XPIProvider.jsm
+++ b/toolkit/mozapps/extensions/internal/XPIProvider.jsm
@@ -2491,7 +2491,7 @@ var XPIProvider = {
         );
       }
       this.maybeInstallBuiltinAddon(
-        "default-theme@mozilla.org",
+        Services.builtInThemes.DEFAULT_THEME_ID,
         "1.3",
         "resource://default-theme/"
       );
