diff --git a/toolkit/mozapps/extensions/content/aboutaddons.js b/toolkit/mozapps/extensions/content/aboutaddons.js
index 6226638b048806fc312882a50775d8a81b749e12..1ab5901851554b240c874f23625bd8a7a04923b5 100644
--- a/toolkit/mozapps/extensions/content/aboutaddons.js
+++ b/toolkit/mozapps/extensions/content/aboutaddons.js
@@ -451,7 +451,7 @@ function nl2br(text) {
  *          The URL of the best fitting screenshot, if any.
  */
 function getScreenshotUrlForAddon(addon) {
-  if (addon.id == "default-theme@mozilla.org") {
+  if (addon.id == Services.builtInThemes.DEFAULT_THEME_ID) {
     return "chrome://mozapps/content/extensions/default-theme/preview.svg";
   }
   const builtInThemePreview = BuiltInThemes.previewForBuiltInThemeId(addon.id);
