diff --git a/tools/mach_commands.py b/tools/mach_commands.py
index f25f5e940cbf98197b79de6a9e7c090a3d32550c..85e6e52d07262622e231d422860a6914f08564e9 100644
--- a/tools/mach_commands.py
+++ b/tools/mach_commands.py
@@ -490,6 +490,29 @@ def npm(command_context, args):
         ensure_exit_code=False,  # Don't throw on non-zero exit code.
     )
 
+@Command(
+    "yarn",
+    category="devenv",
+    description="Run the yarn executable from the NodeJS used for building.",
+)
+@CommandArgument("args", nargs=argparse.REMAINDER)
+def yarn(command_context, args):
+    from mozbuild.nodeutil import find_yarn_executable
+
+    # Avoid logging the command
+    command_context.log_manager.terminal_handler.setLevel(logging.CRITICAL)
+
+    yarn_path, _ = find_yarn_executable()
+
+    if not yarn_path:
+        exit(-1, "could not find npm executable")
+
+    return command_context.run_process(
+        [yarn_path] + args,
+        pass_thru=True,  # Avoid eating yarn output/error messages
+        ensure_exit_code=False,  # Don't throw on non-zero exit code.
+    )
+
 
 def logspam_create_parser(subcommand):
     # Create the logspam command line parser.
